monitorix.cgi

```diff
@@ -448,6 +436,8 @@ if($RRDs::VERSION > 1.2) {
 	if($RRDs::VERSION >= 1.3) {
 		push(@version12_small, "--font=DEFAULT:0:Mono");
 	}
+
+
```

Comment:
Please, remove these blank lines that are left after you removed the extra RRDtool arguments.

Reply:
done